source/AndroidXProject.cshtml

```diff
@@ -1,24 +1,15 @@
 ﻿@using System.Linq
-<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+<Project Sdk="Xamarin.Legacy.Sdk">
   <PropertyGroup>
-    <TargetFramework>MonoAndroid90</TargetFramework>
+    <TargetFrameworks>MonoAndroid90;net6.0-android</TargetFrameworks>
```

Comment:
Week or so I discussed with @Redth about adding MonoAndroid100 and MonoAndroid110 too. His opinion was that it might be OK to do so in the future.

Here is the PR: https://github.com/xamarin/AndroidX/pull/235/files

OK. Fixes for 11.0 are necessary: https://github.com/xamarin/AndroidX/pull/235#issuecomment-754092457

Reply:
You might do that in a separate PR, this one is still waiting on:
1. Xamarin.Legacy.Sdk needs to be on NuGet.org
2. Need to update the "how to build" instructions showing what you need to install: specific builds of Xamarin.Android are required for Xamarin.Legacy.Sdk to work...

Then I would probably need to check with you and @jpobst to test it out some to make sure it works for you locally. 👍

Reply:
Should we start with moving over to `Xamarin.Legacy.Sdk` and only targeting `MonoAndroid90`, and getting that working first?

Reply:
Xamarin.Legacy.Sdk depends on .NET 6, so if it only targeted `MonoAndroid90`, that would still mean:
* You need specific builds of Xamarin.Android, .NET 6, and the Android workload to build.
* The only IDE that works right now is VS Windows 16.9 Preview 4... Hopefully VS Mac is coming soon.

That is why I'm wondering if we really need a `preview` branch to merge this. It could occasionally merge in the latest from master, but would produce new .NET 6 preview packages.